### ISSUE TITLE
Module 'GoogleMapsBase' not found

### ISSUE THREAD

**Issue by benediktveith** [surialabs/react-native-geo-fencing#1]
Receiving this Error 'Module 'GoogleMapsBase' not found' in XCode, using GoogleMaps(2.0.1). 

Googlemaps already made a workaround fix for this Error [here](https://github.com/googlemaps/google-maps-ios-utils/commit/9a5b138d3fb7f70f29d5de86989c65866b65a8ee).

So my workaround fix was to add the compiler flag '-fno-modules' manually to the Pod Target react-native-geo-fencing in Build Settings. I guess adding s.compiler_flags = '-fno-modules' to the podspec file should fix this too.

Building on Android also slightly differ from your Readme... 
Step 3: You need to make the changes to MainApplication not MainActivity.

**Comment by aestrro** [comment]
I'm having this same issue. Are you saying add this in Other Linker Flags?

**Comment by benediktveith** [comment]
@aestrro 
Yes you need to add '-fno-modules' to the Custom Compiler Flags in the Pod Target 'react-native-geo-fencing'
